R/importNGWMN_wml2.R

```diff
@@ -146,6 +148,9 @@ importWaterML2 <- function(input, asDateTime=FALSE, tz="UTC") {
   
   gmlID <- xml_attr(returnedDoc,"id") #TODO: make this an attribute
   TVP <- xml_find_all(returnedDoc, ".//wml2:MeasurementTVP")#time-value pairs
+  if(length(TVP) == 0) { #empty nodes on some sites
```

Comment:
To follow the pattern of other `dataRetrieval` functions, could you return a data.frame with 0 rows, but the columns that are normally populated? I'm guessing:
```
[1] "source"   "date"     "time"     "dateTime" "value"   
[6] "uom"      "comment" 
```